rosetta-client/src/lib.rs

```diff
@@ -53,6 +58,8 @@ pub enum Blockchain {
     Avalanche,
     /// Base
     Base,
+    /// Anvil
```

Comment:
Anvil is not a blockchain. I'd better leave the connector as is, and make required changes in evm backend, specifically [here](https://github.com/Analog-Labs/timechain/blob/a076aaa7065d8497f2a1b463806d263edabd0a11/gmp/evm/src/lib.rs#L407) to pass in a private key